modules/cms-kit/src/Volo.CmsKit.Admin.Application/Volo/CmsKit/Admin/Blogs/BlogPostAdminAppService.cs

```diff
@@ -89,9 +89,10 @@ public virtual async Task<PagedResultDto<BlogPostListDto>> GetListAsync(BlogPost
     {
         var blogs = (await BlogRepository.GetListAsync()).ToDictionary(x => x.Id);
 
-        var blogPosts = await BlogPostRepository.GetListAsync(input.Filter, input.BlogId, input.MaxResultCount, input.SkipCount, input.Sorting);
+        var blogPosts = await BlogPostRepository.GetListAsync(input.Filter, input.BlogId, input.AuthorId,
+            input.MaxResultCount, input.SkipCount, input.Sorting);
 
-        var count = await BlogPostRepository.GetCountAsync(input.Filter);
+        var count = await BlogPostRepository.GetCountAsync(input.Filter, input.BlogId, input.AuthorId);
```

Comment:
@enisn I changed here to filter by blog and user since it is already filtered in GetListAsync function. Is it how it should work?

Reply:
It seems good to work, all those parameters are optional and won't be included if they're not sent. It's a good way as I see